bite-size-notes/week-5-e-SCvx-and-GuSTO-SCP.md

```diff
@@ -0,0 +1,127 @@
+# Successive Convexification (SCvx) and Guaranteed Sequential Trajectory Optimization (GuSTO) SCP Algorithms
+
+## SCOPE
+This note is all about two specific **Sequential Convex Programming (SCP)** algorithms; **Successive Convexification (SCvx)** and **Guaranteed Sequential Trajectory Optimization (GuSTO)**. We will briefly introduce SCP before diving into the two algorithms discussing what 
+they are used for and how to use them. This is a collection of notes taken from a paper written by researchers from the University of 
+Washington's Autonomous Controls Laboratory (UW ACL), the developers of the two algorithms[1].
+
+## Motivation and Introduction
```

Comment:
The motivation and Introduction section has a clear structure and effectively uses diagrams, making them visually easy to understand.